specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/examples/CosmosDBSqlStoredProcedureCreateUpdate.json

```diff
@@ -0,0 +1,37 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2015-04-08",
+    "subscriptionId": "subid",
+    "databaseName": "databaseName",
+    "containerName": "containerName",
+    "storedProcedureName": "storedProcedureName",
+    "createUpdateSqlStoredProcedureParameters": {
+      "properties": {
```

Comment:
This doesn't appear to adhere to the expectations around resource CRUD - you're missing "id", "name" and "type" properties, and the shape of the properties on the PUT doesn't match the shape of the properties on the response.